functions.photon.php

```diff
@@ -10,6 +10,9 @@
  * @return string The raw final URL. You should run this through esc_url() before displaying it.
  */
 function jetpack_photon_url( $image_url, $args = array(), $scheme = null ) {
+	if ( 'image_downsize' !== current_filter() ) {
+		l( $image_url );
```

Comment:
We'll probably need to remove this at some point. It's causing the tests to fail right now:
```
PHP Fatal error:  Call to undefined function l() in /tmp/wordpress-master/src/wp-content/plugins/jetpack/functions.photon.php on line 14
```